backend/compact-connect/stacks/persistent_stack/user_email_notifications.py

```diff
@@ -0,0 +1,74 @@
+from aws_cdk.aws_iam import ServicePrincipal
+from aws_cdk.aws_ses import EmailIdentity, Identity, ConfigurationSet, EmailSendingEvent, EventDestination
+from aws_cdk.aws_sns import Topic, Subscription, SubscriptionProtocol
+from aws_cdk.aws_route53 import TxtRecord, IHostedZone
+from aws_cdk.aws_kms import IKey
+
+from constructs import Construct
+
+
+class UserEmailNotifications(Construct):
+    """
+    This Construct leverages SES to set up an email notification system to send cognito user events from our custom
+    domain with necessary SPF, DKIM, and DMARC verification records.
+
+    The topic is set up to forward all bounce and complaint events to the provided email address.
+    """
+    def __init__(
+            self, scope: Construct, construct_id: str, *,
+            hosted_zone: IHostedZone,
+            environment_context: dict,
+            master_key: IKey,
+            **kwargs
+    ):
+        super().__init__(scope, construct_id, **kwargs)
+
+        domain_name = hosted_zone.zone_name
+        operation_email = environment_context['notifications']['ses_feedback_forwarding_email']
+
+
+        self.email_feedback_topic = Topic(self, "FeedbackTopic",
+                                          display_name="Email Feedback Forwarding Topic",
+                                          master_key=master_key,
+                                          enforce_ssl=True,
+                                          )
+        Subscription(self, "FeedbackSubscription",
+                     topic=self.email_feedback_topic,
+                     protocol=SubscriptionProtocol.EMAIL,
+                     endpoint=operation_email)
+
+        self.config_set = ConfigurationSet(self, "ConfigSet")
+
+        self.config_set.add_event_destination(id="EmailFeedbackEventDestination",
+                                         destination=EventDestination.sns_topic(self.email_feedback_topic),
+                                         configuration_set_event_destination_name="EmailFeedbackDestination",
```

Comment:
Why force this resource name?

Reply:
ah, that's a copilot suggestion that we really don't need. I'll take that out.